src/charm.py

```diff
@@ -66,19 +69,32 @@ def _clone_metrics_repo(self):
         logger.info(f"Cloning {METRICS_REPO}")
         subprocess.check_call(
             [
+                "sudo",
+                "-H",
+                "-u",
+                "ubuntu",
                 "git",
                 "clone",
                 METRICS_REPO,
-                os.path.expanduser("/srv/ubuntu-release-metrics/"),
+                CLONE_LOCATION,
             ],
             env=self.subprocess_env,
         )
+        subprocess.check_call(
```

Comment:
Why is this symlink needed? That sounds a bit like legacy to me, to keep an former location as a symlink to the new one. It's the kind of thing that stays forever, so if it's possible to get rid of it right now, that would probably be best. I expect this would require some other changes elsewhere though.